config/initializers/subject_enqueue_config.rb

```diff
@@ -0,0 +1,18 @@
+module Panoptes
+  module SubjectEnqueue
+    def self.config
+      @config ||= begin
+                    file = Rails.root.join('config/subject_enqueue_config.yml')
+                    YAML.load(File.read(file))[Rails.env].symbolize_keys
+                  rescue Errno::ENOENT, NoMethodError
+                    {  }
+                  end
+    end
+
+    def self.congestion_opts
+      config[:congestion_opts].symbolize_keys
```

Comment:
Rescuing and defaulting to an empty hash in `config` won't help much if you end up calling symbolize_keys on nil here.